fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -373,6 +372,17 @@ public FedoraResource getContainer() {
     }
 
     @Override
+    public FedoraResource getTimeMap() {
```

Comment:
This method only appears to be used once: in the [FedoraResourceImplIT](https://github.com/fcrepo4/fcrepo4/pull/1260/files#diff-0e3024e0f228c42d1ad0611f251af6c4R1143). Therefore, it can probably be removed.

Reply:
@awoods as this defined in the interface I think it must remain here...no?

Reply:
Now that this is used for deleting the TimeMap, I'll assume this comment is resolved.